TF_Settings_Web/src/Components/Pages/Camera/Masking/createCameraData.ts

```diff
@@ -45,18 +44,22 @@ export const updateCameraCanvas = (
     renderScene();
 };
 
+const width = 384;
+const halfWidth = width / 2;
+const doubleWidth = width * 2;
+
```

Comment:
Could we move these constants to the top of the file? I got a bit confused when I saw line 29 without `width` defined above it